The-SMIRNOFF-force-field-format.md

```diff
@@ -1,103 +1,288 @@
+
+## Encodings
+
+Currently, SMIRNOFF supports an XML encoding, which provides a human- and machine-readable form for encoding the parameter set and its typing rules.
+By convention, XML-encoded SMIRNOFF parameter sets are designated with a `.ffxml` extension if written to a filesystem.
+
+## Reference implementation
+
+A reference implementation of the SMIRNOFF XML specification is provided in the [`openforcefield` toolkit](http://github.com/openforcefield/openforcefield).
+
+## Support by molecular simulation packages
+
+While designed for [`OpenMM`](http://openmm.org), parameterized systems created by this toolkit can be converted for use in other popular molecular dynamics simulation packages via [`ParmEd`](http://parmed.github.io/ParmEd) and [`InterMol`](https://github.com/shirtsgroup/InterMol).
 
 ## Basic structure
 
 The SMIRNOFF format provides XML `ffxml` files that are parseable by the `ForceField` class of the `openforcefield.typing.smirnoff` module.
 These encode parameters for a force field based on a SMIRKS-based specification of the chemical environment the parameters are to be applied to.
-The file has tags corresponding to OpenMM force terms (`HarmonicBondForce`, `HarmonicAngleForce`, `PeriodicTorsionForce`, etc., as discussed in more detail below); these specify units used for the different constants provided for individual force terms, for example (see the [AlkEthOH example ffxml](https://github.com/openforcefield/openforcefield/blob/master/openforcefield/data/forcefield/Frosst_AlkEthOH.ffxml)):
+The file has tags corresponding to OpenMM force terms (`BondForce`, `AngleForce`, `TorsionForce`, etc., as discussed in more detail below); these specify units used for the different constants provided for individual force terms, for example (see the [AlkEthOH example ffxml](https://github.com/openforcefield/openforcefield/blob/master/openforcefield/data/forcefield/Frosst_AlkEthOH.ffxml)):
 ```XML
-   <HarmonicAngleForce angle_unit="degrees" k_unit="kilocalories_per_mole/radian**2">
+<AngleForce angle_unit="degrees" k_unit="kilocalories_per_mole/radian**2">
+   ...
+</AngleForce>     
 ```
 which introduces following `Angle` terms which will use units of degrees for the angle and kilocalories per mole per square radian for the force constant.
 
+**QUESTION:** Can we change `smirks` to `smarts` in the attributes, since these are really SMARTS strings augmented with atom tagging (which seems to be broadly supported)?
+
 Under each of these force terms, there are tags for individual parameter lines such as these:
 ```XML
+<AngleForce angle_unit="degrees" k_unit="kilocalories_per_mole/radian**2">
    <Angle smirks="[a,A:1]-[#6X4:2]-[a,A:3]" angle="109.50" k="100.0"/>
    <Angle smirks="[#1:1]-[#6X4:2]-[#1:3]" angle="109.50" k="70.0"/>
+</AngleForce>     
 ```
 The first of these specifies the `[a,A:1]-[#6X4:2]-[a,A:3]` SMIRKS pattern for an angle, with a tetravalent carbon at the center with single bonds to two atoms of any type.
```

Comment:
Can you explicitly say what "a" and "A" mean here, just like defining variables in an equation? "...where `a` means, `A` means, `#6` means element carbon, `:2` means bonded to two other atoms..." Unless you want to set a prerequisite of already knowing SMARTS.